### DISCUSSION TITLE
can we use rtabmap for outdoor side walk and street mapping and localization

### DISCUSSION THREAD

**Discussion by damavand1** [introlab/rtabmap#1080] [General]
can we use rtabmap for outdoor side walk and street mapping and localization ?
and what are the limitations ?

tnx

**Comment by matlabbe** [comment]
Yes, for limitations is greatly depending on the sensors you are using. See https://introlab.3it.usherbrooke.ca/mediawiki-introlab/images/7/7a/Labbe18JFR_preprint.pdf
